packages/mc-scripts/src/commands/login.ts

```diff
@@ -18,27 +63,111 @@ async function run() {
     return;
   }
 
-  console.log(`Enter the login credentials:`);
+  if (shouldUseExperimentalIdentityAuthFlow) {
+    const open = await import('open');
 
-  const { email } = await prompts({
-    type: 'text',
-    name: 'email',
-    message: 'Email',
-  });
-  const { password } = await prompts({
-    type: 'invisible',
-    name: 'password',
-    message: 'Password (hidden)',
-  });
+    const state = generateRandomHash();
+    const nonce = generateRandomHash();
 
-  if (!email || !password) {
-    throw new Error(`Missing email or password values. Aborting.`);
-  }
+    const authUrl = new URL('/login/authorize', mcApiUrl);
+    authUrl.searchParams.set('response_type', 'id_token');
+    authUrl.searchParams.set('response_mode', 'form_post');
+    authUrl.searchParams.set('client_id', '__local:@@cli@@');
```

Comment:
Normally we use that to send the application identifier. The prefix `__local` is actually the important part, as we determine the callback URL (e.g. localhost vs mc hostname). This way we are "backwards compatible" with the OIDC flow from MC apps.